src/sql/src/statements.rs

```diff
@@ -397,6 +400,22 @@ pub fn concrete_data_type_to_sql_data_type(data_type: &ConcreteDataType) -> Resu
     }
 }
 
+pub fn sql_location_to_grpc_add_column_location(
+    location: &Option<AddColumnLocation>,
+) -> Option<api::v1::add_column::Location> {
+    match location {
+        Some(AddColumnLocation::First) => Some(Location {
+            location_type: LocationType::First.into(),
+            after_cloumn_name: "".to_string(),
+        }),
+        Some(AddColumnLocation::After { column_name }) => Some(Location {
+            location_type: LocationType::After.into(),
+            after_cloumn_name: column_name.to_string(),
+        }),
+        None => None,
```

Comment:
The same suggestion, list all possible branches.

Reply:
You may have to introduce greptime-proto dependency in common-query crate and impl `From<AddColumnLocation>` for `add_column::Location` instead of impl `From<Option<AddColumnLocation>>` for `Option<add_column::Location>`

Reply:
Use `impl From<AddColumnLocation> for add_column::Location`